classes/range.js

```diff
@@ -29,10 +29,11 @@ class Range {
     // First reduce all whitespace as much as possible so we do not have to rely
     // on potentially slow regexes like \s*. This is then stored and used for
     // future error messages as well.
```

Comment:
probably worth looking up why the comment has made it into the source (was there a bench or sth?), and if the reason no longer makes sense today, probably worth removing.

Reply:
The code's functionality has not really changed since the comment was added in #564 by @lukekarrys. As such I'm not able to comment. However, I think investigating it falls somewhat outside the scope of this pull request.

Reply:
My intention was rather to figure out if we should remove it, because it says "...so we do not have to rely on potentially slow regexes like \s*". In this PR we switch to a regex, so the comment feels slightly out of place.

Reply:
The previous version also used a similar regex:

```js
this.raw = range
  .trim()
  .split(/\s+/)
  .join(' ')
```

It was introduced in the same pull request as the comment: https://github.com/npm/node-semver/pull/564/files#diff-95bd101ea814ee8011aedabd5c1d1b1a468dbfbf774168e3c3e319a332deadb6R29-R35

Reply:
Ah, I think I got it! The same pull request introduces this change to the regular expressions used elsewhere in the library: https://github.com/npm/node-semver/pull/564/files#diff-7e884a3dd192e2887678190dd2acabc002cd4ecb2e7b5801d33ffa5248b98302R12-R28

That's why the whitespaces need to get collapsed, and why `\s*` regexps can be later avoided.

Reply:
As such I think the comment still holds, as it's not referring to the `\s+` regexp used immediate below it (?)